### ISSUE TITLE
replace run_cmd/run_cmd_qa by vsc-base's run

### ISSUE THREAD

**Issue by boegel** [easybuilders/easybuild-framework#581]
We should replace the mess that is `run_cmd` and `run_cmd_qa` by the (better) equivalent provided by `vsc-base`, i.e. `run`, see https://github.com/hpcugent/vsc-base/blob/master/lib/vsc/utils/run.py

**Comment by mboisson** [comment]
live output would be very nice to have....

**Comment by JensTimmerman** [comment]
This is actually pretty easy to do, I once did it just for testing on my own copy of EB:
https://github.com/JensTimmerman/easybuild-framework/commit/aa3f091b86fed6126fe5cf8d6fe3bd13cd896e09
and https://github.com/JensTimmerman/easybuild-framework/commit/ab1bd70057b8816aa9e4fb1e07afd156bbcf07e5

But I never made a PR because this does pull in a dependency on vsc-base

**Comment by boegel** [comment]
@JensTimmerman Migrating to the `run` support provided by `vsc-base` is still something we should try and do, but we should try and do that without breaking backward compatibility, e.g. `run_cmd` should be a wrapper around one of the `run` functions provided by `vsc.utils.run`?

**Comment by JensTimmerman** [comment]
Yes indeed, I was trying to illustrate how a few minor changes can make a real big change in user experience when running --debug to screen. So yes, please incorporate something similar into EB.

**Comment by Micket** [comment]
So run_cmd got replaced with run_shell_cmd in 5.0.x., so the original issue opened here is no longer relevant. 

But perhaps we should take note of the live request here (which I assume just mean to have the stdout/stderr piped directly to the screen?)
